assets/js/components/ExecutionResults/ExecutionResults.test.jsx

```diff
@@ -1,60 +1,82 @@
 import React from 'react';
+import { screen } from '@testing-library/react';
+
 import { renderWithRouter } from '@lib/test-utils';
 import {
+  addCriticalExpectation,
```

Comment:
Is there any reason not to put the `executions` under the factories import. All of them are exported in the index file.
`@lib/test-utils/factories`